src/pages/Lend/LoanChannelCategoryControl.vue

```diff
@@ -547,6 +555,22 @@ export default {
 			FLSS_ONGOING_EXP_KEY,
 			'EXP-VUE-FLSS-Ongoing-Sitewide'
 		);
+
+		// MP-346 New Add To Basket
+		const newAddToBasketExpData = this.apollo.readFragment({
+			id: `Experiment:${NEW_ADD_TO_BASKET_EXP}`,
+			fragment: experimentVersionFragment,
+		}) || {};
+
+		this.newAddToBasketExpVersion = newAddToBasketExpData.version;
+
+		if (this.newAddToBasketExpVersion) {
+			this.$kvTrackEvent(
```

Comment:
Do we want to rather track the experiment assignment from the header component, since that will run for all pages?